docs/docs/ref-04-tags-and-attributes.md

```diff
@@ -65,10 +65,10 @@ formNoValidate formTarget frameBorder headers height hidden high href hrefLang
 htmlFor httpEquiv icon id inputMode integrity is keyParams keyType kind label
 lang list loop low manifest marginHeight marginWidth max maxLength media
 mediaGroup method min minLength multiple muted name noValidate nonce open
-optimum pattern placeholder poster preload profile radioGroup readOnly rel
-required reversed role rowSpan rows sandbox scope scoped scrolling seamless
-selected shape size sizes span spellCheck src srcDoc srcLang srcSet start step
-style summary tabIndex target title type useMap value width wmode wrap
+optimum pattern placeholder playsInline poster preload profile radioGroup 
+readOnly rel required reversed role rowSpan rows sandbox scope scoped scrolling 
+seamless selected shape size sizes span spellCheck src srcDoc srcLang srcSet 
+start step style summary tabIndex target title type useMap value width wmode wrap
```

Comment:
Can you actually revert the docs change? You have the right idea, we just try to do it in batches around releases (we can update all translations all at once more easily that way)